### PR TITLE
Update Timepicker

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#66487](https://central.tri.be/issues/66487)

### CODE REVIEW

File: src/admin-views/events-meta-box.php [hunk 1]
```diff
@@ -69,62 +61,78 @@
...
+
+							<input
+								autocomplete="off"
+								tabindex="<?php tribe_events_tab_index(); ?>"
+								type="text"
+								class="tribe-timepicker tribe-field-start_time"
+								name="EventStartTime"
+								id="EventStartTime"
+								<?php echo Tribe__View_Helpers::is_24hr_format() ? 'data-format="H:i"' : '' ?>"
+								data-step="<?php echo $start_timepicker_step; ?>"
```
Comment (borkweb) [inline src/admin-views/events-meta-box.php]: `esc_attr()` on data-step and data-round

File: src/admin-views/events-meta-box.php [hunk 2]
```diff
@@ -69,62 +61,78 @@
...
+							<span class="tribe-datetime-separator"> <?php echo esc_html_x( 'to', 'Start Date Time "to" End Date Time', 'the-events-calendar' ); ?> </span>
+
+							<input
+								autocomplete="off"
+								type="text"
+								class="tribe-timepicker tribe-field-end_time"
+								name="EventEndTime"
+								id="EventEndTime"
+								<?php echo Tribe__View_Helpers::is_24hr_format() ? 'data-format="H:i"' : '' ?>"
+								data-step="<?php echo $end_timepicker_step; ?>"
```
Comment (borkweb) [inline src/admin-views/events-meta-box.php]: `esc_attr()` on data-step and data-round here as well.

File: src/resources/js/events-admin.js [hunk 3]
```diff
@@ -1,3 +1,89 @@
+( function ( $, my ) {
```
Comment (borkweb) [inline src/resources/js/events-admin.js]: @barryhughes had a good suggestion of using `obj` rather than `my`.

**Comment by bordoni** [comment]
Rdy for another Look

**Comment by borkweb** [comment]
:+1:

**Pull request merged**